Sources/TuistDependencies/DependenciesController.swift

```diff
@@ -191,20 +191,29 @@ public final class DependenciesController: DependenciesControlling {
 
 extension TuistCore.DependenciesGraph {
     public func merging(with other: Self) throws -> Self {
-        let mergedExternalDependencies = try other.externalDependencies.reduce(into: externalDependencies) { result, entry in
-            if let alreadyPresent = result[entry.key] {
-                throw DependenciesControllerError.duplicatedDependency(entry.key, alreadyPresent, entry.value)
-            }
+        let mergedExternalDependencies = try other.externalDependencies.reduce(into: externalDependencies) { result1, entry1 in
 
-            result[entry.key] = entry.value
+            let platform = entry1.key
+
+            let mine = externalDependencies[platform] ?? [:]
+            let other2 = other.externalDependencies[platform] ?? [:]
```

Comment:
ℹ️ let's try to use meaningful names. Also, isn't other2 entry1.value?
```suggestion
            let platformDependencies = externalDependencies[platform] ?? [:]
            let otherPlatformDependencies = entry1.value
```

Reply:
Fixed namings